book.rb

```diff
@@ -0,0 +1,11 @@
+require './rental'
+
+class Book
+  attr_accessor :title, :author, :rental
```

Comment:
- [ ] Kindly correct your type error here. It should be `rentals`, with the 's'.